core/modules/layout/layout.admin.inc

```diff
@@ -2500,7 +2501,7 @@ function layout_settings_page($form, &$form_state) {
     '#type' => 'submit',
     '#value' => t('Save configuration'),
   );
-
+  backdrop_set_title(t('Layout settings'));
```

Comment:
Could you move this up to the top of the function? It's conventional to put it near the top if using any `backdrop_add_js/css()` or other calls that affect the page.